test/powershell/engine/ParameterBinding/ParameterBinding.Tests.ps1

```diff
@@ -293,20 +293,23 @@
             $date
         }
 
-        # Explict type cast use 'InvariantCulture' by default
-        $expected = [datetime] "1/11/1111"
+        # Explict type cast using 'InvariantCulture'
+        $expected = [System.Management.Automation.LanguagePrimitives]::ConvertTo("1/11/1111", [datetime], [cultureinfo]::InvariantCulture)
 
         # Cast result with ru-RU culture
         $ruCulture = [cultureinfo]::GetCultureInfo("ru-RU")
         $ruCastResult = [System.Management.Automation.LanguagePrimitives]::ConvertTo("1/11/1111", [datetime], $ruCulture)
 
         # Implict parameter argument conversion should also use 'InvariantCulture'
-        $csharpCmdlet, $scriptFunction, $scriptCmdlet = @(
+        $csharpCmdlet, $scriptFunction, $scriptCmdlet = try {
+            $oldCulture = [CultureInfo]::CurrentCulture
             [CultureInfo]::CurrentCulture = 'ru-RU'
```

Comment:
We use the 'ru-Ru' string in line 300 too - maybe put it in a variable?

Reply:
Fixed.